packages/date/src/dateRangeElement.ts

```diff
@@ -0,0 +1,68 @@
+import { html, LitElement, property, TemplateResult } from 'lit-element';
+import { formatDistance, formatISO, formatRelative, Locale } from 'date-fns';
+import { enGB } from 'date-fns/locale';
+import parseISO from 'date-fns/parseISO';
+
+export type DateRangeVariant = 'distance' | 'relative';
+
+export type WeekDay = 0 | 1 | 2 | 3 | 4 | 5 | 6;
+
+export interface DateRangeElementProps {
+  addSuffix?: boolean;
+  baseDate?: string;
+  date: string;
+  includeSeconds?: boolean;
+  locale?: Locale;
+  variant?: DateRangeVariant;
+  weekStartsOn?: WeekDay;
+}
+
+export class DateRangeElement extends LitElement implements DateRangeElementProps {
+  @property({ type: Boolean })
+  addSuffix?: boolean = false;
+
+  @property({ type: String })
+  baseDate: string = formatISO(new Date());
+
+  @property({ type: String })
+  date: string = formatISO(new Date());
+
+  @property({ type: Boolean })
+  includeSeconds?: boolean = false;
+
+  @property({ type: Object })
+  locale: Locale = enGB;
+
+  @property({ type: String })
+  variant: DateRangeVariant = 'relative';
+
+  @property({ type: Number })
+  weekStartsOn: WeekDay = 1;
```

Comment:
rename to lowercase, like `offset`

Reply:
if the attribute is `offset` and the property has an description which explains that it is the week day offset.